src/loaders/loadGeoJsonFeatureCollections.js

```diff
@@ -17,7 +17,7 @@ export const loadGeoJsonFeatureCollections = async (map, styleFunction, groupByP
     const geojsonLayer = L.geoJSON(geojsonData, {
       filter: filterByProperty(groupByProperty, value),
       style: styleFunction(value),
-    }).addTo(map);
```

Comment:
Reasonably if non-breaking. Seems unrelated to the hashing.

Reply:
Thanks @lyret!

I wanted to centralize the definition of what layers are visible on page load to one place, here in `createMap()`:
https://github.com/willtonkin/borderland-map/blob/add-layers-to-url-hash/src/map.js#L43

`loadGeoJsonFeatureCollections()` is only called by `createMap()`, which fires on page load to load  `Placement` and `Placement_map`.

`.addTo(map)` adds the layers to the map, making them visible.

My hope would be to move us towards only using this line to add layers on page load:
https://github.com/willtonkin/borderland-map/blob/master/src/map.js#L178

For this to be the case, we would need to remove all other calls to `.addTo(map)`

However, since the behavior is to always load `Placement` and `Placement_map` regardless of whether they're in the hash `layer` array or not, this PR doesn't change that.

I'm happy to walk this back if needed, as the behavior should be the same with or without the line.

I'll await your feedback @lyret before merging 😊

Here is an additional related change where another `.addTo(map)`  was removed for the same reason:
https://github.com/theborderland/map/pull/8/files#diff-49388a0ca466f7c0f2b6502d32934815a6fad08beb14688f86e75aabd035ca9dL82

Reply:
Ah, I just spotted you'd approved the change so I will merge. Thanks! 🙇🏻

Reply:
Thank you @willtonkin :) fantastic explanation. This seems very reasonable to me, a lot of the front end code was done was rushed to get it out done before the borderland and any help refactoring is much appreciated. We are loosely talking about how to move the project forward for next year - would love to have you join these talks if you’re interested in joining?

Ping @TimGremalm @Gusbod

Reply:
I would love to collab on this! 🤓  I've dropped you a DM in Discord to connect